modules/stochastic_tools/include/utils/GaussianProcessHandler.h

```diff
@@ -103,12 +106,12 @@ class GaussianProcessHandler
    * @param training_data The training data (y values) for the inversion of the
    *                      covariance matrix.
    * @param tao_options Additional options for TAO.
-   * @param show_tao Switch to show details of TAO optimization.
+   * @param show_tao_adam Switch to show details of TAO optimization.
    */
   PetscErrorCode tuneHyperParamsTAO(const RealEigenMatrix & training_params,
                                     const RealEigenMatrix & training_data,
                                     std::string tao_options = "",
-                                    bool show_tao = false);
+                                    bool show_tao_adam = false);
```

Comment:
Maybe changing this parameter to something like: `verbose` is a good idea. I feels odd to have adam in the argument name of a tao-based function.